paperback.lua

```diff
@@ -35,4 +36,37 @@ for key, enabled in pairs(CONFIG) do
         require(SMODS.current_mod.path .. "/" .. path) -- name files "joker_<name>" so they get loaded automatically
         sendDebugMessage("Paperback :: Loaded joker: " .. key)
     end
-end
\ No newline at end of file
+end
+
+
+
+-- Code to allow Ghost Cola to show up
+-- in the shop When Diet Cola is sold
+--------------------------------------------------
+
+-- Creates the flag
+local BackApply_to_run_ref = Back.apply_to_run
+function Back.apply_to_run(arg_56_0)
```

Comment:
These functions can be moved into the top scope of the ghost_cola.lua to keep the code clean, as it has the same effect

Reply:
All of the bottom code in paperback.lua. Git didnt let me select the lines properly